### ISSUE TITLE
TM 4.14+ GM_info BUG

### ISSUE THREAD

**Issue by xinggsf** [Tampermonkey/tampermonkey#1452]
Tested in TM 4.13 and it worked.

## Expected Behavior
```
Uncaught (in promise) TypeError: 'ownKeys' on proxy: trap result did not include 'GM_info'
    at Function.getOwnPropertyNames (<anonymous>)
```

## Specifications

- Chromium: 86; Firefox 98; edge 97
- TM: 4.14+
- add ad's filter rule: `||player.cntv.cn/h5vod/$script`
- test URL: http://tv.cctv.com/2021/09/29/VIDErf36NcgeWYnGqDUJFn7O210929.shtml
- win8.1 x64

## Script
https://gitee.com/xinggsf/gm/raw/master/cntv.user.js

**Comment by derjanb** [comment]
Duplicate of #1416

**Comment by derjanb** [comment]
Should be fixed at 4.16.6158 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|[xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))

Chrome/Edge users, please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions`.
Firefox users please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons` 

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.

**Comment by xinggsf** [comment]
fix bug!! OK
